.github/workflows/job-postcommit-java-validatesrunner-flink-java11.yml

```diff
@@ -0,0 +1,87 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# To learn more about GitHub Actions in Apache Beam check the CI.md
+
+name: Java VR Flink Java 11 PostCommit
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 */6 * * *'
+  push:
+    branches: ['master', 'release-*']
+    tags: ['v*']
+permissions: read-all
+
+jobs:
+  set-properties:
+    runs-on: [self-hosted, ubuntu-20.04]
+    outputs:
+      properties: ${{ steps.test-properties.outputs.properties }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          persist-credentials: false
+      - id: test-properties
+        uses: ./.github/actions/setup-default-test-properties
+
+  java_vr_flink_java11:
+    needs: set-properties
+    name: Run Flink ValidatesRunner Java 11
+    runs-on: [self-hosted, ubuntu-20.04]
+    strategy:
+      fail-fast: false
+      matrix:
+        flink-versions: ${{fromJson(needs.set-properties.outputs.properties).JavaTestProperties.FLINK_VERSIONS}}
+        exclude: # exclude other versions, we only need 1.13 for this test
+          - flink-versions: "1.14"
+          - flink-versions: "1.15"
+    timeout-minutes: 270
+    steps:
+      - name: Get number of available processors
+        id: get_available_processors
+        run: |
+          CORES=$(grep -c ^processor /proc/cpuinfo)
+          BALANCE_CORES=$((3*CORES))
+          echo $BALANCE_CORES
+          echo "::set-output name=BALANCE_CORES::$BALANCE_CORES"
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          persist-credentials: false
+      - name: Setup Environment
+        uses: ./.github/actions/setup-self-hosted-action
+        # This step is run with Java 8 HOME version (default in setup self-hosted actions)
+      - name: Run Flink Test Jar
+        shell: bash
+        run: |
+            ./gradlew :runners:flink:${{ matrix.flink-versions }}:jar
+            sleep 5
```

Comment:
Why do we need this sleep?

Reply:
Hi @damccorm, this is just a precaution, it is not necessary to sleep the process, do you think it would be better to remove it?

Reply:
Hi @damccorm 
would u think it is better to remove it?

Reply:
Lets follow up on this after we make a decision in https://github.com/apache/beam/pull/23292 since its the same issue

Reply:
Fix applied @damccorm